DigitalLearningSolutions.Web/Styles/trackingSystem/progress.scss

```diff
@@ -0,0 +1,7 @@
+@import "~nhsuk-frontend/packages/core/all";

```

Comment:
Do we need to import this here? We don't want to include extra imports as it creates duplicate css when it's built using the current system. The nhs styles are included in the layout.